src/routes/__layout.svelte

```diff
@@ -10,22 +10,32 @@
   import { fetchCluster } from '$lib/services/cluster-service';
   import { fetchNamespaces } from '$lib/services/namespaces-service';
   import { getDefaultNamespace } from '$lib/utilities/get-namespace';
+  import type { GetClusterInfoResponse } from '$types';
 
   export const load: Load = async function ({ url, fetch }) {
     const settings: Settings = await fetchSettings({ url }, fetch);
+    const user = await fetchUser(fetch);
 
-    const { namespaces }: ListNamespacesResponse = await fetchNamespaces(
-      settings,
-      fetch,
-    );
+    let defaultNamespace: string;
+    let cluster: GetClusterInfoResponse;
+    let namespaces: ListNamespacesResponse;
 
-    const defaultNamespace = getDefaultNamespace({ namespaces, settings });
-    const user = await fetchUser(fetch);
-    const cluster = await fetchCluster(settings, fetch);
+    if (!settings.auth.enabled || user.email) {
+      namespaces = await fetchNamespaces(settings, fetch);
+
+      defaultNamespace = getDefaultNamespace({
+        namespaces: [],
```

Comment:
We still want to pass in the namespaces here instead of []

Reply:
good catch